### PR TITLE
Static build

### PR DESCRIPTION
**Pull request by Skunnyk**
This fix the static build of the extension inside php

### CODE REVIEW

File: ext/config.m4 [hunk 1]
```diff
@@ -2,8 +2,20 @@ PHP_ARG_WITH(cassandra, Enable Cassandra extension,
...
 
 AC_MSG_CHECKING([for supported PHP version])
-PHP_CASSANDRA_FOUND_PHP_VERSION=`${PHP_CONFIG} --version`
+if test -z "$PHP_VERSION"; then
+  if test -z "$PHP_CONFIG"; then
+    AC_MSG_ERROR([php-config not found])
+  fi
+  PHP_CASSANDRA_FOUND_PHP_VERSION=`$PHP_CONFIG --version`
+else
+  PHP_CASSANDRA_FOUND_PHP_VERSION=`echo "$PHP_VERSION"`
```
Comment (mikefero) [inline ext/config.m4]: No reason to `echo` here; simply assign the `PHP_CASSANDRA_FOUND_PHP_VERSION` variable.

File: ext/php_driver.h [hunk 2]
```diff
@@ -460,6 +460,9 @@ php5to7_string_compare(php5to7_string s1, php5to7_string s2)
 
 extern zend_module_entry php_driver_module_entry;
 
+#define cassandra_module_ptr &php_driver_module_entry
+#define phpext_cassandra_ptr cassandra_module_ptr
```
Comment (mikefero) [inline ext/php_driver.h]: Could you please update `#define cassandra_module_ptr &php_driver_module_entry` => **`#define phpext_cassandra_ptr &php_driver_module_entry`** and then remove **`#define phpext_cassandra_ptr cassandra_module_ptr`**.

**Review by mikefero — CHANGES_REQUESTED** [review]
Thanks for the contribution and nice catch! Just a couple of requested changes.

**Comment by Skunnyk** [comment]
Thank you for the review. 
PR updated

**Comment by mikefero** [comment]
Thank you for the quick turnaround. The failing tests on TravisCI are nothing to worry about right now; a fix for that will come in later in the week.

Squashing and merging PR.

**Pull request merged**